app.js

```diff
@@ -1499,6 +1499,162 @@ app.delete("/room/:roomId", function (req, res) {
     });
 });
 
+
+//suggest meeting times given list of participants
```

Comment:
Let's update the swagger doc with the expected endpoint and request and response schemas for this endpoint.

Reply:
Will do.